.travis.yml

```diff
@@ -109,8 +108,8 @@ before_install:
       export PATH=$PATH:/tmp/λ
       export PATH=/usr/lib/ccache:$PATH
     else
-      brew update
-      brew install python3 libpng ffmpeg imagemagick mplayer ccache
+      ci/travis/silence brew update
```

Comment:
Is there not a quiet flag or similar that could be passed to `brew update`?

Reply:
I couldn't find any. For example, https://docs.brew.sh/Manpage.html lists only `--merge` and `--force` options.